projects/demo/src/modules/components/wysiwyg/examples/1/index.html

```diff
@@ -0,0 +1,13 @@
+<p>
+    <button tuiButton type="button" (click)="onClick()">Set value</button>
+</p>
+
+<tui-wysiwyg class="editor" exampleText="exampleText" [formControl]="control">
+    Placeholder
+</tui-wysiwyg>
+
+<h4>HTML:</h4>
+<tui-editor-socket [content]="control.value"></tui-editor-socket>
```

Comment:
Maybe let's add second selector matcher `tui-wysiwyg-socket` and deprecate old one?